packages/walt-docs/src/app.js

```diff
@@ -64,7 +72,8 @@ class Explorer extends React.Component {
       requestAnimationFrame(() => {
         try {
           this.intermediateRepresentation = getWaltIR(this.state.code);
-          const wasm = debugWalt(this.intermediateRepresentation);
+          // double parse :(
```

Comment:
And for each frame?

Reply:
Each frame? Only when the code is compiled. I was a bit lazy here, I need to use the parsed AST as the input for the generator.